sourcegraph/cody
# DX: fix dev-desktop watch mode

Because of the updated order of the commands, the `--watch` flag didn't propagate properly to `esbuild`.

## Test plan

1. `pnpm --filter cody-ai run watch:build:dev:desktop` on main
2. update any file in the VS Code folder
3. see that the extension was not rebuilt
4. should be fixed on this branch
